.github/workflows/update-samples.yml

```diff
@@ -0,0 +1,24 @@
+name: Update samples release version
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  update-samples-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Get release version'
+        id: release-version
+        uses: pozetroninc/github-action-get-latest-release@master
```

Comment:
Can we replace this with the commit sha to avoid accidental breakages.

Reply:
Ok